include/seq_macros.inc

```diff
@@ -564,6 +557,16 @@
     .byte \c
 .endm
 
+.macro layer_note1_voice a, b, c
+#ifndef DISABLE_MARIO_PITCH_SHIFTING
+    .byte 0x40 + \a
+#else
+    .byte 0x40 + 39
+#endif
+    var \b
+    .byte \c
+.endm
+
```

Comment:
the voice macros should use the regular macros directly, not duplicate them. also why was portamento moved for no reason

Reply:
Portamento's location made no sense whatsoever, being randomly squeezed between a bunch of unrelated channel commands. I just moved it to the spot it should be.

Reply:
ok but that is basically a formatting change 😔

Reply:
There's literally no other case in which this will ever get moved if this PR is insufficient lol

Reply:
not worried about being moved in the context of hacker alone, but in terms of other mergey stuff. imo keeping changes small and relevant is a good practice for long-term

Reply:
(cozies told me i was allowed to be as nitpicky as i want when i became a dev 😎 )

Reply:
I mean in general completely pointless formatting changes should be avoided, but I think this one is warranted. The change makes sense in the context I'm working with, and the original location of the macro was frankly terrible. Much more justifiable than adding brackets to a random equation or something lmao.

Reply:
hm okay, just change the new macros to use the old ones then

Reply:
Already did 😎